src/jukebox/components/player/core/__init__.py

```diff
@@ -106,22 +120,30 @@ def prev(self):
     @plugin.tag
     def play(self):
         self._active.play()
+        self.player_status.update(playing=True)
```

Comment:
Looks rather annoying to add `self.player_status.update(playing=True/False)` everywhere. You could use a decorator approach, maybe even in the BackendPlayer class (to be verified).

We already use decorators with `@plugin.tag`.